contracts/src/interfaces/IEverlongPositions.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+interface IEverlongPositions {
+    struct Position {
+        /// @dev Checkpoint time when the position matures.
+        uint128 maturityTime;
+        /// @dev Quantity of bonds in the position.
+        uint128 bondAmount;
+    }
+
+    /// @notice Emitted when a new position is added.
+    event PositionAdded(
+        uint128 indexed maturityTime,
+        uint128 bondAmount,
+        uint256 index
+    );
+
+    /// @notice Emitted when an existing position's `bondAmount` is modified.
+    event PositionUpdated(
+        uint128 indexed maturityTime,
+        uint128 newBondAmount,
+        uint256 index
+    );
```

Comment:
I feel like something like `Deposit` and `Withdrawal` might be clearer.

Reply:
I think I misread this initially. Since this is referring to bonds, why don't we just call it `IEverlongBonds` or something like that? This would make it immediately clear what we're talking about.

Reply:
I'm worried that the "bond" wording might be a bit generic and get confusing when the interaction logic for opening/closing longs gets included (seeming like this will be meaty enough to be worth encapsulating in its own contract). 

Was thinking this interface + associated contract would cover higher level and aggregate actions and information on Everlong's bond positions.

Maybe something like `IEverlongFundManagement` or `IEverlongLiquidityManagement` would be appropriate and a bit more specific?

Reply:
If we're worried about it being generic, "Fund," "Liquidity," and "Portfolio" are all more generic/less concrete than "Bond" in this context. I hear what you're saying though since the word "Long" is different than "Bond". Let's just stick with "Position"